examples/mock/cluster_test.go

```diff
@@ -0,0 +1,40 @@
+package test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+	"go.mongodb.org/atlas-sdk/v20231115008/mockadmin"
+)
+
+func TestListClusters(t *testing.T) {
+	// Create mock API.
+	clusterAPI := mockadmin.NewClustersApi(t)
```

Comment:
I think this is very good example on how to use mockery but does not fit to example of SDK mocking.

Mocking example should use "go.mongodb.org/atlas-sdk/v20231115008/admin" interface that is replaced by mocked implementation.

Reply:
Let's make follow up PR.

Reply:
not sure I understand, we have import "go.mongodb.org/atlas-sdk/v20231115008/admin" and the admin interface is being used in the tests:
func MyFunctionCallingListClusters(clusterAPI admin.ClustersApi) (int, error)